bin/pint

```diff
@@ -150,22 +151,94 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        info = False
+        no_info = ''
+        data_json = {}
+        data_xml = None
+        for reg in region.split(','):
+            data = ifsrequest.get_image_data(
+                framework,
+                image_state,
+                output_format,
+                reg.strip(),
+                command_args['--filter']
+            )
+            if '<images/>' in data or '"images": []' in data:
+                info = False
+                continue
+
+            info = True
+            # there is info in some region/s
+            # so we show the empty response/s
+            # and not the message
+            if output_format in output_options:
+                if 'json' in output_format:
+                    data = json.loads(data)
+                    data_json['images'] = data_json.get('images', []) + data.get('images')
+                    data = data_json
+                else:
+                    if data_xml is None:
+                        data_xml = etree.Element('images')
```

Comment:
`data_xml` can be initialized to an etree outside of the loop without considerable cost. The size of an etree element is 64 bytes

```
from lxml import etree
import sys
data_xml = etree.Element('images')
sys.getsizeof(data_xml)
64
```

I don't think we have to worry about those 64 bytes when we have just pulled a lot more data over the wire that we are holding in memory.